tests/test_internals/test_frame_slice_segmentation.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+
+import numpy as np
+from hdmf.testing import TestCase
+from numpy.testing import assert_array_equal
+from parameterized import parameterized, param
+
+from roiextractors.testing import generate_dummy_segmentation_extractor
+
+
+def test_frame_slicing_segmentation_times():
+    num_frames = 10
+    timestamp_shift = 7.1
+    times = np.array(range(num_frames)) + timestamp_shift
+    start_frame, end_frame = 2, 7
+
+    toy_segmentation_example = generate_dummy_segmentation_extractor(num_frames=num_frames, num_rows=5, num_columns=4)
+    toy_segmentation_example.set_times(times=times)
+
+    frame_sliced_segmentation = toy_segmentation_example.frame_slice(start_frame=start_frame, end_frame=end_frame)
+    assert_array_equal(
+        frame_sliced_segmentation.frame_to_time(
+            frames=np.array([idx for idx in range(frame_sliced_segmentation.get_num_frames())])
+        ),
+        times[start_frame:end_frame],
+    )
+
+
+def segmentation_name_function(testcase_function, param_number, param):
+    return f"{testcase_function.__name__}_{param_number}_{parameterized.to_safe_name(param.kwargs['name'])}"
+
+
+class BaseTestFrameSlicesegmentation(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.toy_segmentation_example = generate_dummy_segmentation_extractor(num_frames=15, num_rows=5, num_columns=4)
```

Comment:
I see that you have some tests for rejected and accepted list. Up to your but the `generate_dummy_segmentation_extractor` has a rejected list argument to generate that:

https://github.com/catalystneuro/roiextractors/blob/b71f17db4d3dadeaa4ae02c806c1e269f03d3d9b/src/roiextractors/testing.py#L54-L65

So you can test the non-default case when `rejected_list != [ ]`

Reply:
Yeah, I could. But I actually have yet to see any format in practice that bothers to save ROI info that has not been accepted. (so far, every single ROI written as series data or image mask is 'accepted' so 'rejected' is always empty). Maybe something to think about and revisit in the future